### PR TITLE
Examples in doc/examples/applications edited using object-oriented Matplotlib style

### PR DESCRIPTION
**Pull request by GParolini**
## Description
Issue #7330 
Edited files in doc/examples/applications.

## Checklist

- A descriptive but concise pull request title
- [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- [Unit tests](https://scikit-image.org/docs/dev/development/contribute.html#testing)
- A gallery example in `./doc/examples` for new features
- [Contribution guide](https://scikit-image.org/docs/dev/development/contribute.html) is followed

## Release note

We use [changelist](https://github.com/scientific-python/changelist) to
compile each pull request into an item of the release notes. Please refer to
the [instructions](https://scikit-image.org/docs/stable/development/contribute.html#documenting-changes)
and [past release notes](https://scikit-image.org/docs/stable/release_notes/index.html)
for guidance and examples.

```release-note
All examples in doc/examples/applications now use object-oriented Matplotlib style.
```

### CODE REVIEW

File: doc/examples/applications/plot_3d_image_processing.py [hunk 1]
```diff
@@ -94,7 +94,7 @@
 
 def show_plane(ax, plane, cmap="gray", title=None):
     ax.imshow(plane, cmap=cmap)
-    ax.axis("off")
+    ax.set_axis_off()
```
Comment (lagru) [inline doc/examples/applications/plot_3d_image_processing.py:97]: I'm curious why you changed some of these but not all? E.g. in [`plot_general.py`](https://github.com/scikit-image/scikit-image/blob/9107d523a899b8d3767630941ac28f438bf875d2/doc/examples/data/plot_general.py#L58)? 
To be clear I don't mind either way, both methods are part of the OO API. But it does blow up the diff to quite a lot. Keeping diffs small can help speeding things along as getting big PRs merged can sometimes be a slog.

The same thing applies with 
- `fig, (ax1, ax2) = plt.subplots(...)` and 
- `fig, axes = plt.subplots(...)` 

which were changed to `fig, ax = plt.subplots(...)`.

File: doc/examples/applications/plot_image_comparison.py [hunk 2]
```diff
@@ -81,7 +85,8 @@
 #
 # `blend` is the result of the average of the two images.
 
-fig = plt.figure(figsize=(8, 9))
+
```
Comment (lagru) [inline doc/examples/applications/plot_image_comparison.py:88]:
```suggestion
```

File: doc/examples/applications/plot_pixel_graphs.py [hunk 3]
```diff
@@ -35,7 +35,7 @@
 _, ax = plt.subplots()
 ax.imshow(retina_source)
 ax.set_axis_off()
-_ = ax.set_title('Human retina')
+ax.set_title('Human retina')
```
Comment (lagru) [inline doc/examples/applications/plot_pixel_graphs.py:38]: I'd revert these kind of changes. The `_` is used to hide the object returned by `ax.set_title` which is unnecessarily shown in the gallery example otherwise.

File: doc/examples/applications/plot_pixel_graphs.py [hunk 4]
```diff
@@ -35,7 +35,7 @@
 _, ax = plt.subplots()
```
Comment (lagru) [inline doc/examples/applications/plot_pixel_graphs.py:35]:
```suggestion
fig, ax = plt.subplots()
```

File: doc/examples/applications/plot_image_comparison.py [hunk 5]
```diff
@@ -33,7 +33,7 @@
 # The `checkerboard` method alternates tiles from the first and the second
 # images.
 
-fig = plt.figure(figsize=(8, 9))
+fig, ax = plt.subplots(figsize=(8, 9))
```
Comment (lagru) [inline doc/examples/applications/plot_image_comparison.py:36]: I think using `fig, ax = plt.subplots` over `fig = plt.figure` is actually more confusing here since `ax` is never used below. `plt.figure` might be fine as long as the returned object -- the figure -- is captured and used...

**Review by lagru — COMMENTED** [review]
Thanks very much for starting the clean-up @GParolini! Though I think you missed a few cases where matplotlib's old API is still in use. You can find these cases like so

```sh
$ grep -R "plt.plot" --include '*.py' doc/
doc/examples/applications/plot_image_comparison.py:plt.plot()
doc/examples/applications/plot_image_comparison.py:plt.plot()
doc/examples/applications/plot_image_comparison.py:plt.plot()
doc/examples/transform/plot_radon_transform.py:    plt.plot(response, label=f)
doc/examples/segmentation/plot_rolling_ball.py:plt.plot(x, label='original')
doc/examples/segmentation/plot_rolling_ball.py:plt.plot(x - background, label='radius=80')
doc/examples/segmentation/plot_rolling_ball.py:plt.plot(x - background2, label='radius=10')
doc/examples/segmentation/plot_hausdorff_distance.py:plt.plot(set_ax, set_ay, 'or')
doc/examples/segmentation/plot_hausdorff_distance.py:plt.plot(set_bx, set_by, 'og')
doc/examples/segmentation/plot_hausdorff_distance.py:plt.plot(x_line, y_line, 'y')
doc/examples/segmentation/plot_hausdorff_distance.py:plt.plot(x_line, y_line, 'y')
doc/source/auto_examples/applications/plot_image_comparison.py:plt.plot()
doc/source/auto_examples/applications/plot_image_comparison.py:plt.plot()
doc/source/auto_examples/applications/plot_image_comparison.py:plt.plot()
doc/source/auto_examples/transform/plot_radon_transform.py:    plt.plot(response, label=f)
doc/source/auto_examples/segmentation/plot_rolling_ball.py:plt.plot(x, label='original')
doc/source/auto_examples/segmentation/plot_rolling_ball.py:plt.plot(x - background, label='radius=80')
doc/source/auto_examples/segmentation/plot_rolling_ball.py:plt.plot(x - background2, label='radius=10')
doc/source/auto_examples/segmentation/plot_hausdorff_distance.py:plt.plot(set_ax, set_ay, 'or')
doc/source/auto_examples/segmentation/plot_hausdorff_distance.py:plt.plot(set_bx, set_by, 'og')
doc/source/auto_examples/segmentation/plot_hausdorff_distance.py:plt.plot(x_line, y_line, 'y')
doc/source/auto_examples/segmentation/plot_hausdorff_distance.py:plt.plot(x_line, y_line, 'y')
```

I'd approve the current state except for a few minor things as many changes are a clear improvement already. :D

**Comment by lagru** [comment]
cc'ing @mkcor in case I misunderstood what is requested in https://github.com/scikit-image/scikit-image/issues/7330.

File: doc/examples/applications/plot_3d_image_processing.py (hunk 1 above)
Comment (GParolini) [reply doc/examples/applications/plot_3d_image_processing.py:97]: I have not made yet to the file plot_general.py. I only changed the files in the folder applications/ so far.
I left (ax1, ax2) and similar in a few instances where I feared to break the docs making a change. E.g. in plot_image_comparison.py, GridSpec is used to arrange the plots in a more complex layout and changing fig, (ax1, ax2, ax3) into fig, ax generated an error I could not solve.

File: doc/examples/applications/plot_pixel_graphs.py (hunk 3 above)
Comment (GParolini) [reply doc/examples/applications/plot_pixel_graphs.py:38]: Ok, thanks for explaining.

File: doc/examples/applications/plot_image_comparison.py (hunk 5 above)
Comment (GParolini) [reply doc/examples/applications/plot_image_comparison.py:36]: I see your point. I must say that this example was quite peculiar due to the use of the GridSpec and I was not really sure how to handle it. It is also the example where I left plt.plot(). But, I guess, also in this example plt.plot can be removed and plt.show() added at the end of the file, as for the other examples in the folder.

File: doc/examples/applications/plot_3d_image_processing.py (hunk 1 above)
Comment (mkcor) [reply doc/examples/applications/plot_3d_image_processing.py:97]: I don't see any reason, in general, for changing `fig, axes = plt.subplots(...)` or `fig, (ax1, ax2) = plt.subplots(...)` into `fig, ax = plt.subplots(...)`. Using `ax1` and `ax2` may even be more convenient and readable than `ax[0]` and `ax[1]`. Anyway, these changes have nothing to do with the OO style vs pyplot-style issue.

File: doc/examples/applications/plot_coins_segmentation.py [hunk 6]
```diff
@@ -17,11 +17,11 @@
 coins = data.coins()
 hist, hist_centers = histogram(coins)
 
-fig, axes = plt.subplots(1, 2, figsize=(8, 3))
-axes[0].imshow(coins, cmap=plt.cm.gray)
-axes[0].axis('off')
-axes[1].plot(hist_centers, hist, lw=2)
-axes[1].set_title('histogram of gray values')
+fig, ax = plt.subplots(1, 2, figsize=(8, 3))
```
Comment (mkcor) [inline doc/examples/applications/plot_coins_segmentation.py:20]: Please revert (see above comment).

File: doc/examples/applications/plot_coins_segmentation.py [hunk 7]
```diff
@@ -33,16 +33,16 @@
 # binary image that either misses significant parts of the coins or merges
 # parts of the background with the coins:
 
-fig, axes = plt.subplots(1, 2, figsize=(8, 3), sharey=True)
+fig, ax = plt.subplots(1, 2, figsize=(8, 3), sharey=True)
```
Comment (mkcor) [inline doc/examples/applications/plot_coins_segmentation.py:36]: idem

File: doc/examples/applications/plot_cornea_spot_inpainting.py [hunk 8]
```diff
@@ -166,7 +166,7 @@ def plot_comparison(plot1, plot2, title1, title2):
 
 footprint = ski.morphology.diamond(3)
 mask_open = ski.morphology.opening(mask_2, footprint)
-plot_comparison(mask_2, mask_open, "mask before", "after opening")
+plot_comparison(mask_2, mask_open, "Mask before", "After opening")
```
Comment (mkcor) [inline doc/examples/applications/plot_cornea_spot_inpainting.py:169]: Why not, but this is unrelated to the current issue, and the PR is very long already, so I would prefer having these changes in a separate/dedicated PR. Thanks!

File: doc/examples/applications/plot_face_detection.py [hunk 9]
```diff
@@ -25,7 +25,7 @@
 is specified in the xml file with trained parameters. The ``scale_ratio``
 parameter specifies by which ratio the search window is increased on each
 step. If you increase this parameter, the search time decreases and the
-accuracy decreases. So, faces on some scales can be not detected.
+accuracy decreases. So, faces on some scales cannot be detected.
```
Comment (mkcor) [inline doc/examples/applications/plot_face_detection.py:28]: Please revert: The meaning is different! The meaning is that, by increasing the step, you take the risk of missing more faces (because there are more scales you don't search).

Comment (GParolini) [reply doc/examples/applications/plot_face_detection.py]: Reverted

File: doc/examples/applications/plot_cornea_spot_inpainting.py (hunk 8 above)
Comment (GParolini) [reply doc/examples/applications/plot_cornea_spot_inpainting.py]: Reverted. In general, though, it would be helpful to know if plot titles should start with a capital letter or not. At present, there is a mix, even within the same example.

File: doc/examples/applications/plot_coins_segmentation.py (hunk 7 above)
Comment (GParolini) [reply doc/examples/applications/plot_coins_segmentation.py]: Reverted

File: doc/examples/applications/plot_coins_segmentation.py (hunk 6 above)
Comment (GParolini) [reply doc/examples/applications/plot_coins_segmentation.py]: Reverted.

File: doc/examples/applications/plot_cornea_spot_inpainting.py (hunk 8 above)
Comment (mkcor) [reply doc/examples/applications/plot_cornea_spot_inpainting.py]: You are absolutely right, @GParolini. It's just that, precisely, this issue deserves its own... issue/PR :wink: 

(see also @lagru's comment about avoiding blowing up the diff :pray:)

File: doc/examples/applications/plot_face_detection.py [hunk 10]
```diff
@@ -45,14 +45,14 @@
...
-false detections.  True faces usually has a lot of detections around them and
+false detections.  True faces usually have a lot of detections around them and
 false ones usually have single detection. First algorithm searches for
 clusters: two rectangle detections are placed in the same cluster if the
 intersection score between them is larger then
 ``intersection_score_threshold``. The intersection score is computed using the
 equation (intersection area) / (small rectangle ratio). The described
 intersection criteria was chosen over intersection over union to avoid a corner
-case when small rectangle inside of a big one have small intersection score.
+case when small rectangle inside of a big one has small intersection score.
```
Comment (mkcor) [inline doc/examples/applications/plot_face_detection.py:55]: This gallery example would definitely need some proofreading... (see also "criteri*a* was chosen" above) This would deserve its own issue/PR as well.

File: doc/examples/applications/plot_fluorescence_nuclear_envelope.py [hunk 11]
```diff
@@ -150,15 +150,15 @@
 image_t_0_channel_1 = image_sequence[0, 1, :, :]
 selection = np.where(mask, image_t_0_channel_1, 0)
 
-fig, (ax0, ax1) = plt.subplots(1, 2, figsize=(12, 6), sharey=True)
+fig, ax = plt.subplots(1, 2, figsize=(12, 6), sharey=True)
```
Comment (mkcor) [inline doc/examples/applications/plot_fluorescence_nuclear_envelope.py:153]: I would revert this, as explained previously, because the current code is correct and already OO style.

File: doc/examples/applications/plot_human_mitosis.py [hunk 12]
```diff
@@ -76,10 +76,10 @@
 fig, ax = plt.subplots(ncols=2, figsize=(10, 5))
 ax[0].imshow(image)
 ax[0].set_title('Original')
-ax[0].axis('off')
+ax[0].set_axis_off()
```
Comment (mkcor) [inline doc/examples/applications/plot_human_mitosis.py:79]: I'd like to ask the same question @lagru did in https://github.com/scikit-image/scikit-image/pull/7346#discussion_r1527178981. Maybe because `set_axis_off()` looks more like `set_title()` and the other `set_` functions (so, visually and mentally, it's nicer/lighter/smoother)? @GParolini I'd be curious if you had other ideas on this!

File: doc/examples/applications/plot_morphology.py [hunk 13]
```diff
@@ -38,13 +38,13 @@
 
 
 def plot_comparison(original, filtered, filter_name):
-    fig, (ax1, ax2) = plt.subplots(ncols=2, figsize=(8, 4), sharex=True, sharey=True)
```
Comment (mkcor) [inline doc/examples/applications/plot_morphology.py:41]: Please revert.

File: doc/examples/applications/plot_pixel_graphs.py (hunk 3 above)
Comment (mkcor) [reply doc/examples/applications/plot_pixel_graphs.py:38]: @lagru I like using `;` for hiding returned objects in Matplotlib, but Sphinx-gallery errors on them :cry:

File: doc/examples/applications/plot_solidification_tracking.py [hunk 14]
```diff
@@ -5,7 +5,7 @@
 
 In this example, we identify and track the solid-liquid (S-L) interface in a
 nickel-based alloy undergoing solidification. Tracking the solidification over
-time enables the calculatation of the solidification velocity. This is
+time enables the calculation of the solidification velocity. This is
```
Comment (mkcor) [inline doc/examples/applications/plot_solidification_tracking.py:8]: Thanks!

File: doc/examples/applications/plot_pixel_graphs.py (hunk 4 above)
Comment (mkcor) [reply doc/examples/applications/plot_pixel_graphs.py:35]: @lagru isn't this unnecessary, if `fig` is never used afterwards? :thinking:

File: doc/examples/applications/plot_human_mitosis.py (hunk 12 above)
Comment (GParolini) [reply doc/examples/applications/plot_human_mitosis.py:79]: Both axis("off") and set_axis_off() were used. I always changed axis("off") in set_axis_off() because all the other axis methods (title, xlim, etc.) use set. It seemed  to me more natural to have the same structure for everything.

File: doc/examples/applications/plot_image_comparison.py (hunk 2 above)
Comment (GParolini) [reply doc/examples/applications/plot_image_comparison.py]: Reverted. plt.plot() is now replaced with plt.show()

File: doc/examples/applications/plot_morphology.py (hunk 13 above)
Comment (GParolini) [reply doc/examples/applications/plot_morphology.py:41]: Reverted

File: doc/examples/applications/plot_fluorescence_nuclear_envelope.py (hunk 11 above)
Comment (GParolini) [reply doc/examples/applications/plot_fluorescence_nuclear_envelope.py]: Reverted

File: doc/examples/applications/plot_image_comparison.py (hunk 5 above)
Comment (GParolini) [reply doc/examples/applications/plot_image_comparison.py]: Reverted

File: doc/examples/applications/plot_pixel_graphs.py (hunk 3 above)
Comment (GParolini) [reply doc/examples/applications/plot_pixel_graphs.py]: Reverted the changes, including ax instead of axes.

**Comment by stefanv** [comment]
> Thanks very much for starting the clean-up @GParolini! Though, I think you missed a few cases where matplotlib's old API is still in use. You can find these cases like so
> 
> ```shell
> $ grep -R "plt.plot" --include '*.py' doc/
> ```

In case you haven't yet, give [ripgrep](https://github.com/BurntSushi/ripgrep) or [ugrep](https://github.com/Genivia/ugrep) a try!

**Comment by stefanv** [comment]
Thanks for helping us make the docs more consistent, @GParolini!

File: doc/examples/applications/plot_3d_image_processing.py (hunk 1 above)
Comment (lagru) [reply doc/examples/applications/plot_3d_image_processing.py:97]: > I only changed the files in the folder applications/ so far.

Ah I should have noticed. Then let's keep this PR focused on only that directory. And we can do follow-ups for the other directories.

I concur with @mkcor but how do we go forward? I'd say to minimize work on all sides let's keep the instances where you changed `.axis("off")` to `.set_axis_off()` but let's not change more. They are easy to review but the consistency is not really a big improvement in this case.

For the `(ax1, ax2)` vs `ax` issue, I also don't see a big difference. Let's keep the ones you changed and not change more going forward. @mkcor agreed?

File: doc/examples/applications/plot_pixel_graphs.py (hunk 4 above)
Comment (lagru) [reply doc/examples/applications/plot_pixel_graphs.py]: It is. I think I thought it might be slightly more confusing to break up the standard `fig, ax = plt.subplots()`. But your are right to call me out. Requesting to revert this is also noise. :sweat_smile:

File: doc/examples/applications/plot_3d_image_processing.py (hunk 1 above)
Comment (GParolini) [reply doc/examples/applications/plot_3d_image_processing.py:97]: Sorry, now I am bit lost about the changes that I still need to do for this pull request and the general guidelines you wish to see implemented. I try to recap:

**For this pull request**

- Change plt.close() in fig.close()
- Change plt.tight_layout() in fig.tight_layout()
- I can revert the set_axis_off()  in axis("off") when this was the original use (just say what you prefer)
- I think I already reverted all fig, ax = plt.subplots() in _, ax = plt.subplots(), but if I will check if I missed any.

**In general**

-  _, ax = plt.subplots() does not need to be changed in fig, ax = plt.subplots()
- fig, axes = plt.subplots() does not need to be changed in fig, ax = plt.subplots()
- fig, (ax1, ax2) = plt.subplots() and similar do not need to be changed in fig, ax = plt.subplots()
- Axis notations ax1, etc. and ax[1], etc. are both acceptable
- set_axis_off()  and axis("off") are both acceptable
- plt.close() should be changed in fig.close()
- plt.tight_layout() should be changed in fig.tight_layout()
- plt.plot() must be replaced by plt.show()
- Editorial changes in docs (capital letters in titles, etc.) are better left for future interventions.

Please provide your feedback.

Comment (mkcor) [reply doc/examples/applications/plot_3d_image_processing.py:97]: Hi both,

@GParolini said (private communication) that she would away in the next days and we could take over the PR. For future reference:

> let's keep the instances where you changed .axis("off") to .set_axis_off() but let's not change more

Agreed.

> For the `(ax1, ax2)` vs `ax` issue, I also don't see a big difference. Let's keep the ones you changed and not change more going forward. @mkcor agreed?

Hmm, I would revert these, since a) it's unrelated to the OO vs pyplot-style and b) `(ax1, ax2)` is correct. I can take care of it (see above).

> Change plt.close() to fig.close()

Yes, for sure, since plt.close() is pyplot-style, so it corresponds to the issue we're addressing with this PR.

> Change plt.tight_layout() in fig.tight_layout()

idem

I don't think I need to go through all the bullet points, since the issue is unambiguous: Everything pyplot-style needs to be changed into OO style, period.

File: doc/examples/applications/plot_coins_segmentation.py [hunk 15]
```diff
@@ -42,7 +42,7 @@
 axes[1].set_title('coins > 150')
 
 for a in axes:
-    a.axis('off')
+    a.set_axis_off()
 
 plt.tight_layout()
```
[the remark this answers is not in the document]
Comment (mkcor) [reply doc/examples/applications/plot_coins_segmentation.py]: Done a89f24198811823.

File: doc/examples/applications/plot_3d_image_processing.py (hunk 1 above)
Comment (mkcor) [reply doc/examples/applications/plot_3d_image_processing.py:97]: > plt.close() should be changed in fig.close()

Apparently this doesn't work (see [failure](https://github.com/scikit-image/scikit-image/actions/runs/8360293702/job/22885635348?pr=7346)); reverted 09df1bc48cfc75da.

File: doc/examples/applications/plot_cornea_spot_inpainting.py [hunk 16]
```diff
@@ -124,11 +124,11 @@
 
 
 def plot_comparison(plot1, plot2, title1, title2):
-    fig, (ax1, ax2) = plt.subplots(ncols=2, figsize=(12, 6), sharex=True, sharey=True)
-    ax1.imshow(plot1, cmap='gray')
-    ax1.set_title(title1)
-    ax2.imshow(plot2, cmap='gray')
-    ax2.set_title(title2)
+    fig, ax = plt.subplots(ncols=2, figsize=(12, 6), sharex=True, sharey=True)
```
Comment (lagru) [inline doc/examples/applications/plot_cornea_spot_inpainting.py:127]: @mkcor, I'm confused? In https://github.com/scikit-image/scikit-image/pull/7346#discussion_r1532120117 you indicate you wanted to revert these yourself?

File: doc/examples/applications/plot_text.py [hunk 17]
```diff
@@ -16,9 +16,9 @@
 
 img = data.cat()
 
-fig = plt.figure()
-fig.figimage(img, resize=True)
-fig.text(0, 0.99, "I am stefan's cat.", fontsize=32, va="top")
+fig, ax = plt.subplots()
+ax.figure.figimage(img, resize=True)
+ax.text(0, 0.99, "I am stefan's cat.", fontsize=32, va="top")
```
Comment (lagru) [inline doc/examples/applications/plot_text.py:21]: Something in these changes apparently breaks the [example](https://output.circle-artifacts.com/output/job/7542931b-2ec6-4606-a94e-e33e10db810f/artifacts/0/doc/build/html/auto_examples/applications/plot_text.html)? There's not text on the image?

Probably because `text()` is now called on `ax` instead. Maybe just revert since `fig = plt.figure()` seems like OO?

```suggestion
fig = plt.figure()
fig.figimage(img, resize=True)
fig.text(0, 0.99, "I am stefan's cat.", fontsize=32, va="top")
```

**Review by lagru — CHANGES_REQUESTED** [review]
One of the examples broke except for that I'd be happy to merge.

**Comment by mkcor** [comment]
> One of the examples broke

Which one? [1b3835e](https://github.com/scikit-image/scikit-image/pull/7346/commits/1b3835e3c46ec5ebaceba2e74d56d04d898e1347) passed all tests... :thinking: 

Why [806d8ea](https://github.com/scikit-image/scikit-image/pull/7346/commits/806d8ea51f65f2d4c5f24c3f574ffdb344c1dff7)? I'm personally not in favour of reverting any `plt.subplots()`.

File: doc/examples/applications/plot_cornea_spot_inpainting.py (hunk 16 above)
Comment (mkcor) [reply doc/examples/applications/plot_cornea_spot_inpainting.py:127]: Right, I forgot to go through them! Sorry. I'll do it now if you're not currently editing the branch...

**Comment by lagru** [comment]
>> One of the examples broke
>
> Which one? [1b3835e](https://github.com/scikit-image/scikit-image/pull/7346/commits/1b3835e3c46ec5ebaceba2e74d56d04d898e1347) passed all tests... 🤔

The one linked in https://github.com/scikit-image/scikit-image/pull/7346#discussion_r1532708822. The text is missing from the image. Also axes isn't needed in the example. Feel welcome to use `subplots` but then probably `fig, _ = plt.subplots(...)`?

File: doc/examples/applications/plot_cornea_spot_inpainting.py (hunk 16 above)
Comment (mkcor) [reply doc/examples/applications/plot_cornea_spot_inpainting.py]: Done ae5b047eb0557301.

**Review by lagru — APPROVED** [review]

**Comment by mkcor** [comment]
> The one linked in [#7346 (comment)](https://github.com/scikit-image/scikit-image/pull/7346#discussion_r1532708822).

Oh, thank you! Somehow I missed that comment.

> The text is missing from the image. Also axes isn't needed in the example. Feel welcome to use `subplots` but then probably `fig, _ = plt.subplots(...)`?

I see. I'll sleep on this and wrap up tomorrow. :pray:

**Comment by mkcor** [comment]
PS: @lagru thank you for double-checking the example *visually* :heart:

**Comment by lagru** [comment]
Thanks @GParolini! :D
